CybridSDKTestAppUITests/Flow/TradeFlow.swift

```diff
@@ -0,0 +1,822 @@
+        }
+        
+        // -- Enter in BTC
+        let usdc = app.staticTexts["USDC (ERC-20) USDC"]
+        if usdc.waitForExistence(timeout: 5) {
+            
+            XCTAssertTrue(usdc.exists)
+            usdc.tap()
+            
+            // -- Check UISegmentedControl
+            let buy = app.buttons["BUY"]
+            if buy.waitForExistence(timeout: 6) {
+                
+                XCTAssertTrue(buy.exists)
+                XCTAssertTrue(app.buttons["SELL"].exists)
+                let segmented = app.segmentedControls.element(boundBy: 0)
+                XCTAssertTrue(segmented.buttons.element(boundBy: 0).isSelected)
+            }
+            
+            // -- Set how many USD to trade
+            let switchButton = app.buttons["TradeComponent_SWitchButton"]
+            if switchButton.waitForExistence(timeout: 5) {
+                switchButton.tap()
+            }
+            
+            let amountField = app.textFields["TradeComponent_AmountField"]
+            if amountField.waitForExistence(timeout: 2) {
+                tapElementAndWaitForKeyboardToAppear(amountField)
+                amountField.typeText(foundsToTrade)
+            }
+            //dismissKeyboardIfPresent()
+            
+            // -- Trigger action (buy/sell)
+            let actionButton = app.buttons["TradeComponent_ActionButton"]
+            if actionButton.waitForExistence(timeout: 2) {
+                actionButton.tap()
+            }
+            
+            // Modal: Confirm
+            
+            // -- Title
+            let confirmTitle = app.staticTexts["Order Quote"]
+            if confirmTitle.waitForExistence(timeout: 6) {
+                XCTAssertTrue(confirmTitle.exists)
+            }
+            
+            // -- Amount
+            let amountTitle = app.staticTexts[foundsToTradeFormatted]
+            if amountTitle.waitForExistence(timeout: 4) {
+                XCTAssertTrue(amountTitle.exists)
+            }
+            
+            // -- Confirm deposit
+            let confirmTrade = app.buttons["Confirm"]
+            if confirmTrade.waitForExistence(timeout: 2) {
+                confirmTrade.tap()
+            }
+            
+            // Modal: Confirm
+            
+            // -- Title
+            let detailsTitle = app.staticTexts["Order Submitted"]
+            if detailsTitle.waitForExistence(timeout: 6) {
+                XCTAssertTrue(detailsTitle.exists)
+            }
+            
+            let confirmTrade_2 = app.buttons["TradeComponent_Modal_Success_ConfirmButton"]
+            if confirmTrade_2.waitForExistence(timeout: 2) {
+                confirmTrade_2.tap()
+            }
+            
+            // -- Return main controller
+            self.returnTap()
+            
+        } else {
+            
+            // -- Return main controller
+            self.returnTap()
+        }
+    }
+    
+    func trade_buy_SOL() {
```

Comment:
For these buy/ sell flows, can we create a helper function to encapsulate all the logic and then just call the helper with the correct asset and action (i.e., Buy or Sell)?